README.md

```diff
@@ -217,9 +220,41 @@ In general, your changelog notes should answer the following questions:
 
 ## Releases 🚀
 
+Taken from https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases:
+
+> Releases are deployable software iterations you can package and make available for a wider audience to download and use.
+Releases are based on Git tags, which mark a specific point in your repository's history.
+A tag date may be different from a release date since they can be created at different times.
+
+GitHub release consists of a git tag, some text (commonly referred to as Release Notes) and artifacts (binaries, executables, any kind of documentation, etc.)
+
+### Tagging and naming releases
+
+Release tags and names consist of the letter `v` followed by a version number.
+
+Below tags are all valid examples:
+* `v0.1.4`
+* `v0.9.3`
+* `v1.0.0`
+* `v1.1.0`
+* `v1.12.45`
+* `v5.0.0`
+
```

Comment:
this defer from what you have written in the beginning of the document about git tags. In the beginning the tags do not have `v` in front. I would agree with that since it is easier to manage in code.

Reply:
after changing this you can merge it yourself

Reply:
essentially as written in the next comment, there is a versioning part and there you do not mention `v` ...